fury/ui/containers.py

```diff
@@ -220,6 +268,73 @@ def re_align(self, window_size_change):
             msg = "You can only left-align or right-align objects in a panel."
             raise ValueError(msg)
 
+    def update_border_coords(self):
+        """Update the coordinates of the borders
+        """
+        self.border_coords = {'left': (0., 0.),
+                              'right': (1., 0.),
+                              'top': (0., 1.),
+                              'bottom': (0., 0.)}
+
+        for key in self.borders.keys():
+            self.update_element(self.borders[key], self.border_coords[key])
+
+    @property
+    def border_color(self):
+        sides = ['left', 'right', 'top', 'bottom']
+        return [self.borders[side].color for side in sides]
+
+    @border_color.setter
+    def border_color(self, side_color):
+        """Set the color of a specific border
+
+        Parameters
+        ----------
+        side_color: Iterable
+            Iterable to pack side, color values
+        """
+        side, color = side_color
+
+        if side.lower() not in ['left', 'right', 'top', 'bottom']:
+            raise ValueError(
```

Comment:
can you add a test for this case?

Reply:
Done with [c43adff](https://github.com/fury-gl/fury/pull/441/commits/c43adff5b8f7f21d539bf58dc8e2f12d4d0cf640).